packages/palette/src/elements/Pagination/LargePagination.tsx

```diff
@@ -84,20 +83,25 @@ const Page: React.FC<PageProps> = props => {
     onClick(cursor, page)
   }
 
+  const highlight = isCurrent ? "black5" : "white100"
+  const href = ""
+
   return (
-    <Button active={isCurrent} onClick={handleClick}>
-      <Sans size="3" weight="medium" display="inline">
-        {page}
-      </Sans>
-    </Button>
+    <Box bg={highlight} borderRadius={2}>
+      <Link href={href} onClick={handleClick} underlineBehavior="hover">
```

Comment:
We've talked about it, but it will be interesting to see how this behaves within the Router. I think, for the first pass, we'll have to pass in an `onClick` handler in force that promptly does 

```
event.preventDefault() 
```

which will cancel the `href` behavior. 

(We can talk more if this doesn't make sense @jonallured)